backend-nodejs/README.md

```diff
@@ -0,0 +1,24 @@
+[Main](../README.md)
+# Webapp Boilerplate (backend)
+This repository goes with [webapp-boilerplate-frontend](https://github.com/josephmfaulkner/webapp-boilerplate-frontend)
```

Comment:
You'll likely want to eliminate all references to your own GitHub account and repositories